tidb-resource-control.md

```diff
@@ -192,6 +192,125 @@ The following example binds the current statement to the resource group `rg1`.
 SELECT /*+ RESOURCE_GROUP(rg1) */ * FROM t limit 10;
 ```
 
+### Manage queries that consume more resources than expected (Runaway Queries)
+
+> **Warning:**
+>
+> This feature is an experimental feature. It is not recommended that you use it in the production environment. This feature might be changed or removed without prior notice. If you find a bug, you can report an [issue](https://github.com/pingcap/tidb/issues) on GitHub.
+
+A runaway query is a query that takes longer to execute or consumes more resources than expected. Starting from TiDB v7.2.0, the resource control feature introduces the management of runaway queries. You can set conditions for a resource group to identify runaway queries and automatically take actions to prevent them from exhausting resources and affecting normal queries.
```

Comment:
```suggestion
A runaway query is a query that takes excessive time or resources than expected. Starting from TiDB v7.2.0, the resource control feature introduces the management of runaway queries. You can set criteria for a resource group to identify runaway queries and automatically take actions to prevent them from exhausting resources and affecting other queries.
```